pkg/metal/core.go

```diff
@@ -361,6 +361,17 @@ func (p *Provider) GetVolumeIDs(ctx context.Context, req *driver.GetVolumeIDsReq
 			continue
 		}
 
+		// use only the nguid part of the volumeHandle for lightbits volumes
+		if spec.CSI.Driver != "csi.lightbitslabs.com" {
```

Comment:
did you mean `==`

Reply:
thanks, this explains why it didn't work as expected :joy: